config/rbac/leader_election_role_binding.yaml

```diff
@@ -8,5 +8,5 @@ roleRef:
   name: leader-election-role
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: controller-manager
```

Comment:
How will update process behave when these are changed? I don't know if we have medusa-operator update process, or do we simply delete the existing one?

Reply:
Honestly, I just dumbly followed the operator sdk upgrade steps that you can find [here](https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.6.1/).
I don't actually know about the update process, but that's something I can experiment to see how it goes.
So we're just talking about using the current version of medusa-operator and then upgrade to this version, right?

Reply:
Well, yes. This isn't related to operator-sdk update, but just normal Kubernetes deployment. You're changing the name of the deployment and serviceAccount. Now, in k8ssandra we don't probably use these so it doesn't matter.